gh-repo-export

```diff
@@ -231,10 +256,36 @@ download_migration() {
   fi
 }
 
+print_archive_url() {
+  local MIGRATION_ID="$1"
+  local MIGRATION_STATE="${MIGRATIONS[$MIGRATION_ID]}"
+
+  if [[ $GH_HOST != "" ]]; then
+    if [[ "$GH_HOST" == "github.com" ]]; then
+      BASE_URL="https://api.github.com"
+    else
+      BASE_URL="https://$GH_HOST/api/v3"
+    fi
+  else
+    BASE_URL="https://api.github.com"
+  fi
```

Comment:
I know something like this is necessary because the GitHub CLI doesn't expose commands that generate these urls based on the host authenticated and selected [here](https://github.com/cli/cli/blob/60f2da2d1f22201b4f624fced9c10731b7c51acb/internal/ghinstance/host.go#L85-L96).

Doesn't mean I have to like it 😠